README.md

```diff
@@ -1,25 +1,33 @@
-# Console Standard
+This repository hosts the [Console Standard](https://console.spec.whatwg.org/).
 
-All web browsers, and most other JavaScript environments, have shipped some form of developer console. This standard attempts to finally define the semantics of the console APIs, in an attempt to create convergence.
+## Code of conduct
 
-A list containing some of the observed differences so far in different environments is available in [NOTES.md](NOTES.md).
```

Comment:
If you want to preserve this you could create a READMEEND.md file that contains this and its own level-2-heading. Thoughts?